modules/dxp/apps/analytics/analytics-reports-web/src/main/resources/META-INF/resources/js/components/detail/KeywordsDetail.js

```diff
@@ -27,7 +26,6 @@ export default function KeywordsDetail({
 				className="mb-2"
 				dataProvider={trafficVolumeDataProvider}
 				label={Liferay.Util.sub(Liferay.Language.get('traffic-volume'))}
-				popoverAlign={ALIGN_POSITIONS.Bottom}
```

Comment:
Just to be sure, is removing this for consistency, or was it some limitation you encountered?

Reply:
We don't need align anymore, it's not needed for the Popover. I think we add it for align the element inside the useLayoutEffect. Now, we just use `alignPosition` ("top", "bottom"), as the API said: https://clayui.com/docs/components/popover/api.html.

Reply:
Oh ok, I thought this was passed to `alignPosition` at `ClayPopover` at the end. Thanks for clarifying!